lib/locales/es.json

```diff
@@ -0,0 +1,434 @@
+        "failedToUpdateGroup": "Error al actualizar el grupo",
+        "adminLabel": "(Administrador)",
+        "member": "Miembro",
+        "removeFromGroupTitle": "¿Remover del grupo?",
+        "removeFromGroupConfirmation": "¿Estás seguro de que quieres remover a {name} del grupo? Perderán acceso a todos los mensajes y actividad del grupo. Tendrás que invitarlos de nuevo si los quieres de vuelta.",
+        "removedFromGroupSuccess": "{name} removido del grupo",
+        "failedToRemoveMember": "Error al remover miembro",
+        "noUserToStartChatWith": "No hay usuario para iniciar chat"
+    },
+    "login": {
+        "signInWithNostrKey": "Iniciar sesión con tu llave de Nostr",
+        "createNewNostrKey": "Crear una nueva llave de Nostr",
+        "signInDescription": "Tu llave será cifrada y almacenada únicamente en tu dispositivo.",
+        "logIn": "Iniciar sesión",
+        "welcomeTo": "Bienvenido a",
+        "slogan": "Seguro. Distribuido. Incensurable."
+    },
+    "clipboard": {
+        "emptyTextError": "No se encontró texto en el portapapeles",
+        "readError": "Error al leer del portapapeles",
+        "error": "Error del portapapeles"
+    },
+    "media": {
+        "downloadError": "Error de descarga",
+        "image": "Imagen"
+    },
+    "auth": {
+        "login": "Iniciar sesión",
+        "signup": "Registrarse",
+        "loginToWhiteNoise": "Iniciar sesión en White Noise",
+        "setupProfile": "Configurar perfil",
+        "beyondTheNoise": "Más allá del ruido",
+        "privacyAndSecurity": "Privacidad y Seguridad",
+        "chooseIdentity": "Elegir identidad",
+        "decentralizedAndPermissionless": "Descentralizado y sin permisos",
+        "createNewNostrKey": "Crear nueva llave de Nostr",
+        "signInWithNostrKey": "Iniciar sesión con llave de Nostr",
+        "scanQrCode": "Escanear código QR",
+        "whiteNoise": "White Noise",
+        "slogan": "Descentralizado. Incensurable.\nMensajería segura.",
+        "chooseAName": "Elige un nombre",
+        "introduceYourself": "Preséntate",
+        "yourName": "Tu nombre",
+        "writeSomethingAboutYourself": "Escribe algo sobre ti",
+        "enterYourPrivateKey": "Ingresa tu llave privada",
+        "qrCodeDetected": "¡Código QR detectado!",
+        "scanYourPrivateKeyQr": "Escanea tu código QR de llave privada.",
+        "pleaseEnterPrivateKey": "Por favor ingresa tu llave privada",
+        "setUpProfile": "Configurar perfil",
+        "finish": "Finalizar",
+        "privacySecuritySubtitle": "Mantén tus conversaciones privadas. Incluso en caso de una brecha, tus mensajes permanecen seguros.",
+        "chooseIdentitySubtitle": "Chatea sin revelar tu número de teléfono o email. Elige tu identidad: nombre real, seudónimo o anónimo.",
+        "decentralizedPermissionlessSubtitle": "Ninguna autoridad central controla tu comunicación—no se necesitan permisos, no es posible la censura."
```

Comment:
`Ninguna autoridad central controla tu comunicación. No se necesitan permisos ni es posible la censura.` A dot instead of - sounds more natural. in spanish to me.